pages/templates/pages/microsite_page.html

```diff
@@ -1,12 +1,210 @@
 {% extends "pages/base_page.html" %}
 
-{% load wagtailcore_tags %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block meta %}
+  {{ block.super }}
+  <style>
+    #custom_header {
+      {% if page.custom_header_background_color %}
+        background-color: #{{ page.custom_header_background_color }};
+      {% else %}
+        background: transparent;
+      {% endif %}
+
+      {% if page.show_accent_border %}
+        border-top: solid 8px #{{page.accent_border_color}};
+      {% endif %}
+    }
+
+    #primary-cta {
+      {% if page.button_background_color %}
+        background-color: #{{page.button_background_color}};
+        border-color: #{{page.button_background_color}};
+      {% endif %}
+
+      {% if page.button_text_color %}
+        color: #{{page.button_text_color}};
+      {% endif %}
+    }
+
+    .microsite-content a#primary-cta {
+      {% if page.button_text_color %}
+        color: #{{page.button_text_color}};
+      {% endif %}
+    }
+
+    /* TODO: Add hover color to model or dynamically generate */
+    #primary-cta:hover {
```

Comment:
Why adding this? Is this not default button behavior?

Reply:
No, default behavior is for hover to be a darker color, but this obviously won't work with custom colors, as we'd have to come up with a solution to calculate the dark color.